src/main/java/org/truffleruby/cext/CExtNodes.java

```diff
@@ -1074,9 +1074,9 @@ public Object debug(Object... objects) {
     public abstract static class HiddenVariableGetNode extends CoreMethodArrayArgumentsNode {
 
         @Specialization(guards = "isRubySymbol(name)")
-        public Object hiddenVariableGet(DynamicObject object, DynamicObject name,
+        public Object hiddenVariableGet(VirtualFrame frame, DynamicObject object, DynamicObject name,
                 @Cached("createObjectIVarGetNode()") ObjectIVarGetNode iVarGetNode) {
-            return iVarGetNode.executeIVarGet(object, name);
+            return iVarGetNode.executeIVarGet(frame, object, name);
```

Comment:
Could you rebase on top of #812 once it's merged?
That should make the diff smaller.